README.md

```diff
@@ -27,10 +27,20 @@ cordova plugin add https://github.com/samzilverberg/cordova-mixpanel-plugin.git
 **window.mixpanel.people:**
 
 - identify(distinctId, onSuccess, onFail)
-- registerPushToken(apnsOrGcmToken, onSuccess, onFail)
+- registerPushId(registrationId, onSuccess, onFail)
```

Comment:
i see what you mean about the parameter name, its more obvious when your using that push plugin.

but i'd still insist on changing the fn name, so its obvious that no 'registering' is taking place here, but only setting of a pre-registered registrationId